Panic upon retaining keys in config-gen

**Describe the bug**
When running `skywire-cli visor gen-config -r --retain-keys`, the following panic happens:

```
INFO [skywire-cli]: No 'output' set; using default path: /home/pi/go/src/github.com/SkycoinProject/skywire-mainnet/skywire-config.json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4d0270]

goroutine 1 [running]:
github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands/visor.fillInOldKeys(0x14dc640, 0x4d, 0x1481440, 0x14814d0, 0x0)
	/home/pi/go/src/github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands/visor/gen-config.go:89 +0x178
github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands/visor.glob..func8(0xb5fa90, 0x140e8f0, 0x0, 0x2)
	/home/pi/go/src/github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands/visor/gen-config.go:70 +0x1c8
github.com/spf13/cobra.(*Command).execute(0xb5fa90, 0x140e8e0, 0x2, 0x2, 0xb5fa90, 0x140e8e0)
	/home/pi/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x1f4
github.com/spf13/cobra.(*Command).ExecuteC(0xb5ecd0, 0x1, 0x4296c, 0x14201b0)
	/home/pi/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x230
github.com/spf13/cobra.(*Command).Execute(...)
	/home/pi/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands.Execute()
	/home/pi/go/src/github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/commands/root.go:28 +0x20
main.main()
	/home/pi/go/src/github.com/SkycoinProject/skywire-mainnet/cmd/skywire-cli/skywire-cli.go:11 +0x14
```

**Environment information:**

- OS: e.g. Linux Raspian
- `develop` branch

--- a/cmd/skywire-cli/commands/visor/gen-config.go
+++ b/cmd/skywire-cli/commands/visor/gen-config.go
@@ -86,8 +86,7 @@ func fillInOldKeys(confPath string, conf *visor.Config) error {
 		return fmt.Errorf("invalid old configuration file: %w", err)
 	}
 
-	conf.KeyPair.PubKey = oldConf.KeyPair.PubKey
-	conf.KeyPair.SecKey = oldConf.KeyPair.SecKey
+	conf.KeyPair = oldConf.KeyPair
 
 	return nil
 }

